src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    public TaskList() {
+        taskList = new ArrayList<>();
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return this.taskList;
+    }
+
+    public Task getTask(int index) {
+        return taskList.get(index);
+    }
+
+    public void printTaskList() {
+        int index = 0;
+        for (Task task: this.taskList) {
+            System.out.println("    " + (++index) + "." + task.toString());
+        }
+    }
+
+    public int size() {
+        return this.taskList.size();
+    }
```

Comment:
I'm not sure whether this is counted as a getter, but if it is, maybe `getSize` may be a more appropriate method name